qutip/qip/gates.py

```diff
@@ -177,6 +177,27 @@ def phasegate(theta, N=None, target=0):
                     dims=[[2], [2]])
 
 
+def qrot(theta, phi, N=None, target=0):
```

Comment:
what about being a bit more explicit and call it `qubit_rotate`, `qrotate, or `rotate_qubit`? Maybe `qrot` is fine too.